README.md

```diff
@@ -240,13 +240,13 @@ dashboards:
 
 ## BASE64 dashboards
 
-Dashboards could be storaged in a server that does not return JSON directly and instead of it returns a Base64 encoded file (e.g. Gerrit)
+Dashboards could be stored on a server that does not return JSON directly and instead of it returns a Base64 encoded file (e.g. Gerrit)
```

Comment:
`in` -> `on` isn't quite a spelling thing in that it isn't a typo, it's just fixing the grammar to use the right word...